client/src/commands/util.ts

```diff
@@ -32,6 +33,24 @@ export function showConnectionQuickPickBox(prompt: string): Thenable<string | un
     return vscode.window.showQuickPick(connectionNames, quickPickOptions);
 }
 
+export function showIdentityConnectionQuickPickBox(prompt: string, connection: any): Thenable<string | undefined> {
```

Comment:
Does this need an associated test?

Reply:
no because it is covered by other tests for addConnectionIdentity